app/src/main/java/com/example/sharingang/NewItemActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.sharingang
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.example.sharingang.databinding.ActivityNewItemBinding
+
+class NewItemActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityNewItemBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_new_item)
+    }
+
+    fun createNewItem(view: View) {
+        val intent = Intent(baseContext, MainActivity::class.java)
+        intent.putExtra(
+            "LAST_DESCRIPTION",
+            binding.description
+        )
```

Comment:
I think it would be cleaner to declare "LAST DESCRIPTION" as a public static final String of NewItemActivity.

Reply:
Indeed.

I think it would be even better to use a standard intent extra key, like `Intent.EXTRA_TEXT`, having read a bit more:
https://developer.android.com/guide/components/intents-filters.

Reply:
Agreed, perfect. For some reason, I cannot find (or I may be blind) the declaration of EXTRA_TEXT, where is it located?